ui/src/app/pages/admin/admin-user-common.tsx

```diff
@@ -221,6 +221,30 @@ export const displayTierBadgeByRequiredModule = (
   );
 };
 
+export const getEraNote = (profile: Profile): string => {
+  const requiredForRT = isEraRequiredForTier(
+    profile,
+    AccessTierShortNames.Registered
+  );
+  const requiredForCT = isEraRequiredForTier(
+    profile,
+    AccessTierShortNames.Controlled
+  );
+  let note = `*eRA Commons requirements vary by institution. This user's institution 
+  (${profile.verifiedInstitutionalAffiliation.institutionDisplayName}) `;
```

Comment:
Some Test users don't have a `verifiedInstitutionalAffiliation` so this causes the page to crash.  See `grushton` for an example.

Reply:
Good catch. Can I use `N/A` if institution name is undefined?
<img width="829" alt="Screen Shot 2022-02-01 at 10 33 50 AM" src="https://user-images.githubusercontent.com/35533885/151999044-10997432-9aef-4618-ad8a-71e2271d0f3d.png">

Reply:
Since we don't have any information about the institution, we should say that.  "the inst doesn't require era" is not quite true.  how about: "We don't have any institutional information for this user"

Reply:
Updated.